src/main/groovy/org/shipkit/internal/gradle/downstream/test/TestDownstreamTask.java

```diff
@@ -10,7 +10,8 @@
 import java.util.List;
 
 import static java.util.Arrays.asList;
-import static org.shipkit.internal.gradle.util.StringUtil.capitalize;
+import static org.apache.commons.lang.StringUtils.capitalize;
```

Comment:
Is this change on purpose?
I'd rather stay with our `StringUtil` in order to minimize dependencies on external libs.